core/src/com/mygdx/crazybomber/model/player/Player.java

```diff
@@ -3,17 +3,21 @@
 import com.mygdx.crazybomber.model.bomb.Bomb;
 
 import java.util.Stack;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
 
 public class Player {
     private boolean _isKnockedUp;
     private boolean _isAlive;
-    private int _numBombs;
     private int _rangeBomb;
```

Comment:
instead of using rangeBomb consider using something like numberOfRangeUpgrades to not confuse between the bomb's rangeBomb and player's rangeBomb

Reply:
did _numRangeUpgrades and adjusted code for it